### PR TITLE
Add a dialog showing all reactions to a message

### PR DESCRIPTION
**Pull request by chillmastonmuutos**
This PR partly implements the features described here: https://github.com/vector-im/element-web/issues/9723

What is NOT implemented as in the above issue:
- Timestamps on hover
- Moderator functionalities

Here's a few screenshots:
Screenshot 2022-03-14 at 12 05 01
Screenshot 2022-03-14 at 12 05 13

*NOTE!* I could not do the sign-off part for this PR, because I do not wish to associate my real name with this GitHub account. I've sent an email to dco@matrix.org regarding anonymous contributions, but so far I've recieved no reply. I'm opening this PR in order to avoid my work going to waste.

---
Here's what your changelog entry will look like:

## ✨ Features
 * Add a dialog showing all reactions to a message ([\#8051](https://github.com/matrix-org/matrix-react-sdk/pull/8051)). Contributed by @chillmastonmuutos.

### CODE REVIEW

File: src/i18n/strings/fi.json
Comment (robintown) [inline src/i18n/strings/fi.json:3207]: Please don't edit the translation files directly (other than `en_EN.json`). Weblate is the source of truth for these files, so you'll need to add the translation there once this PR is merged.

**Review by robintown — DISMISSED** [review]

File: src/i18n/strings/fi.json
Comment (gsouquet) [inline src/i18n/strings/fi.json:3207]:
```suggestion
    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta"
```

**Review by gsouquet — CHANGES_REQUESTED** [review]
Thank you for your contribution, in order to move this pull request forward it will need to go through design review. I've added the design team to the list of reviewers, alternatively you can join the [Element Design](https://matrix.to/#/#element-design:matrix.org) room on Matrix.

Before I go through a more in-depth code review, I believe this would require some tests

File: src/i18n/strings/fi.json
Comment (chillmastonmuutos) [reply src/i18n/strings/fi.json]: Thanks! Removed the Finnish translation.

Comment (chillmastonmuutos) [reply src/i18n/strings/fi.json]: Thanks for the input! I can add some tests if you are happy with how the feature works at the moment.

**Comment by chillmastonmuutos** [comment]
Tests added

**Comment by turt2live** [comment]
@chillmastonmuutos when you get a chance, can you update this PR to latest develop for us?

**Review by turt2live — CHANGES_REQUESTED** [review]
(this is blocked on a check on that DCO form submission)

**Comment by turt2live** [comment]
Hi @chillmastonmuutos - our legal department says while they have received your DCO signoff request, they do not have it marked completed. 

Until that is completed we won't be able to accept this change, sorry. If you have questions, please forward them to the legal team using the same address you sent the DCO request to.

**Comment by t3chguy** [comment]
@hanthor needs a designer to take it on

**Comment by t3chguy** [comment]
Thanks for your contribution. I'm going to close this for now as the changes requested haven't been made. If you're interested in continuing to work on this please let us know and we can reopen the pull request. Thanks!

**Comment by joerglohrer** [comment]
Please reopen this, as it's still not resolved and strongly needed!

**Pull request closed without merging**
